### ISSUE TITLE
Support separate connections for databases in one connection.

### ISSUE THREAD

**Issue by LonwoLonwo** [dbeaver/dbeaver#17263]
Sometimes we have a connection with a server with some databases inside.
Some databases can be read in one connection - like MSSQL.
But other databases exist, which demand separate connections for each database.
Without it, we can't read or change metadata in non-default databases.

We did it for PostgreSQL.

And we should add this opportunity for Azure or Netezza.

**Comment by keyrox** [comment]
Yes please, this is such a big deal for us working with Azure Databases.

**Comment by triforcely** [comment]
Hey @LonwoLonwo , is this fix on a roadmap somewhere?

**Comment by mayanknamasys20** [comment]
Hi Team, is there any plan about by when this feature will be implemented?

**Comment by JeVuSopra** [comment]
Hi, 
Is there any update on this feature?
Thanks in advance :)

**Comment by HariTrigger** [comment]
Hi guys, does this feature includes Oracle too? 
Thanks in advance,
